geode-core/src/main/java/org/apache/geode/distributed/ConfigurationProperties.java

```diff
@@ -1921,46 +1921,43 @@
   /**
-   * The static String definition of the <i>"redis-bind-address"</i> property <a
-   * name="redis-bind-address"/a>
+   * The static String definition of the <i>"compatible-with-redis-bind-address"</i> property <a
+   * name="compatible-with-redis-bind-address"/a>
    * </p>
-   * <U>Description</U>: Specifies the address on which the Redis API for Geode is listening. If set
-   * to the empty string or this property is not specified, the server listens on all local
-   * addresses.
+   * <U>Description</U>: Specifies the address on which Geode APIs compatible with Redis is
+   * listening. If set to the empty string or this property is not specified, the server listens on
+   * all local addresses.
    * </p>
    * <U>Default</U>: ""
    */
-  String REDIS_BIND_ADDRESS = "redis-bind-address";
+  String REDIS_BIND_ADDRESS = "compatible-with-redis-bind-address";
   /**
-   * The static String definition of the <i>"redis-enabled"</i> property <a
-   * name="redis-enabled"/a>
+   * The static String definition of the <i>"compatible-with-redis-enabled"</i> property <a
+   * name="compatible-with-redis-enabled"/a>
    * </p>
-   * <U>Description</U>: When the default value of false, the Redis API for Geode is not available.
-   * Set to true to enable the Redis API for Geode.</td>
+   * <U>Description</U>: When the default value of false, the Geode APIs compatible with Redis is
+   * not available.
+   * Set to true to enable the Geode APIs compatible with Redis.</td>
    * </p>
    * <U>Default</U>: false
-   * <td>redis-enabled</td>
-   * <td>When the default value of false, the Redis API for <%=vars.product_name%> is not available.
-   * Set
-   * to true to enable the Redis API for <%=vars.product_name%>.</td>
-   * <td>S</td>
-   * <td>false</td>
-   * </tr>
+   * When the default value of false, the Geode APIs compatible with Redis is not available.
+   * Set to true to enable the Geode APIs compatible with Redis.
    * </p>
    */
-  String REDIS_ENABLED = "redis-enabled";
+  String REDIS_ENABLED = "compatible-with-redis-enabled";
   /**
-   * The static String definition of the <i>"redis-password"</i> property <a
-   * name="redis-password"/a>
+   * The static String definition of the <i>"compatible-with-redis-password"</i> property <a
+   * name="compatible-with-redis-password"/a>
    * </p>
    * <U>Description</U>: Specifies the password that the server uses when a client attempts to
    * authenticate.
    * </p>
    * <U>Default</U>: no password set
    */
-  String REDIS_PASSWORD = "redis-password";
+  String REDIS_PASSWORD = "compatible-with-redis-password";
   /**
-   * The static String definition of the <i>"redis-port"</i> property <a name="redis-port"/a>
+   * The static String definition of the <i>"compatible-with-redis-port"</i> property <a
+   * name="compatible-with-redis-port"/a>
    * </p>
    * <U>Description</U>: Specifies the port on which the server listens for Redis API for Geode
```

Comment:
It seems like you should also change this from "Redis API for Geode" to something mentioning the "the Geode APIs compatible with Redis" as you did above (if it's important)